### ISSUE TITLE
Video Transformations help needed?

### ISSUE THREAD

**Issue by apalumbo** [marnusw/cloudinary-tiny-js#4]
Hi, we are currently using cloudinary and to be honest the filesize issue is something really bad.
Do you need any help to implement the video transformations? We would love to switch to that lib.

**Comment by marnusw** [comment]
@apalumbo thank you so much for the offer to help! We are still using this library on various projects, but as you can see it hasn't been updated in ages because I just don't have the time.

If you need video transformation support then yes, by all means, please add it and submit a PR. We could even discuss giving you access to the repo in time if you want to stay involved long-term.

Since the point of this lib is small bundle size I think it is important that someone who only needs image transformations don't have to import the video related code as well. I recall I had a plan for how that should work, but I think was messy and right now it seems have two main exports from the library, one for images and one for video, might be the best way to go.

If you feel that things are over complicated in the way the configuration is handled it probably is and I'm open to a refactor. It's something I've known was due for a while.

I also took great care to validate the input to the various transformation functions, but I've been thinking using TypeScript one could probably get rid of a lot of the manual type checking, so if you wanted to add the video bits in TS I would support that.

In short, I will appreciate any help on this library and you can to a large extent have free reign. 

Please ping me on email if you think I might have missed a future issue or PR, I don't always catch them all.

**Comment by marnusw** [comment]
I've switched the library to TypeScript and simplified the setup a bit. Feel free to contribute video transformations along the same lines as images if you are still interested. It will be appreciated.

**Comment by marnusw** [comment]
I will close this because there is an invitation to contribute video transformations in the readme and there is little point having this open until a PR is submitted.
